src/main/kotlin/frc/robot/subsystems/PickAndPlaceSubsystem.kt

```diff
@@ -0,0 +1,170 @@
+        var elbowVelocity = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElbowVelocity").publish()
+        var elevatorPosition = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElevatorPosition").publish()
+        var elevatorVelocity = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElevatorVelocity").publish()
+
+        val elevatorVoltage = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElevatorVoltage").publish()
+        val bottomHit = NetworkTableInstance.getDefault().getTable("Arm").getBooleanTopic("BottomHit").publish()
+        val topHit = NetworkTableInstance.getDefault().getTable("Arm").getBooleanTopic("TopHit").publish()
+        val wristPosition = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("WristPosition").publish()
+        val wristVoltageNT = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("WristVoltage").publish()
+        val elbowVoltageNT = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElbowVolts").publish()
+        val elevatorVoltageNT = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElevatorVolts").publish()
+        val intakeVoltageNT = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("IntakeVolts").publish()
+
+    }
+
+
+    init {
+        //intake
+        wristMotor.idleMode = CANSparkMax.IdleMode.kBrake
+        intakeOneMotor.idleMode = CANSparkMax.IdleMode.kBrake
+        intakeTwoMotor.idleMode = CANSparkMax.IdleMode.kBrake
+        elbowMotor.idleMode = CANSparkMax.IdleMode.kBrake
+        elevatorMotor.idleMode = CANSparkMax.IdleMode.kBrake
+
+        intakeOneMotor.inverted = IntakeConstants.intakeMotorRInverted
+        intakeTwoMotor.inverted = IntakeConstants.intakeMotorLInverted
+
+
+        intakeOneMotor.setSmartCurrentLimit(IntakeConstants.intakeMotorsCurrentLimit)
+        intakeTwoMotor.setSmartCurrentLimit(IntakeConstants.intakeMotorsCurrentLimit)
+        wristMotor.setSmartCurrentLimit(IntakeConstants.pitchMotorCurrentLimit)
+
+        intakeOneMotor.burnFlash()
+        intakeTwoMotor.burnFlash()
+        wristMotor.burnFlash()
+        //everything else:
+        // TODO: Elevator conversion factors have been tuned, but the elbow conversion factors have not.
+        elbowEncoder.positionConversionFactor = 2.0 * Math.PI / 60.0
+        elbowEncoder.velocityConversionFactor = 2.0 * Math.PI / 60.0
+
+        elevatorEncoder.positionConversionFactor = ArmConstants.elevatorEncoderPositionConversionFactor
+        elevatorEncoder.velocityConversionFactor = ArmConstants.elevatorEncoderVelocityConversionFactor
+
+        wristEncoder.positionConversionFactor = 2.0 *Math.PI
+        wristEncoder.velocityConversionFactor = 2.0 *Math.PI / 60.0
+        // TODO: wristEncoder position and velocity conversion factor
+    }
+
+    val elbowPositionRadians get() = elbowEncoder.position + ArmConstants.elbowEncoderPosOffset
```

Comment:
Go to KCMT and get the actual getters for these, they have already been made. Made need to be tuned, but still a good start.